netdata.spec.in

```diff
@@ -447,12 +447,20 @@ ${RPM_BUILD_DIR}/%{name}-%{version}/packaging/bundle-ebpf.sh ${RPM_BUILD_DIR}/%{
 
 %pre
 
-# User/Group creations, as needed
-getent group netdata >/dev/null || groupadd -r netdata
-getent group docker >/dev/null || groupadd -r docker
-getent passwd netdata >/dev/null || \
-  useradd -r -g netdata -G docker -s /sbin/nologin \
-    -d %{contentdir} -c "netdata" netdata
+if ! getent group netdata > /dev/null; then
+  groupadd --system netdata
+fi
+
+if ! getent passwd netdata > /dev/null; then
+  useradd --system -g netdata --home %{contentdir} --no-create-home -s /sbin/nologin -c "Netdata pseudo user" netdata
```

Comment:
What dir is `%{contentdir}`? Can you show `getent passwd netdata` after installing?

Reply:
%{contentdir} is defined in the start of this script. As you can see, it's been used in earlier version of this function. I just used it, though I'm not sure in its usefulness.

Reply:
What do you mean by 'show after installing'? In the script?

Reply:
`/var/lib/netdata` should be the correct home directory in this case.